maraboupy/MarabouNetwork.py

```diff
@@ -17,12 +17,12 @@
 '''
 
 from maraboupy import MarabouCore
-from maraboupy import MarabouUtils
 from maraboupy.MarabouPythonic import *
```

Comment:
This import could be removed.

`from maraboupy.MarabouPythonic import *`

Reply:
Fixed.